front/app/containers/Admin/formBuilder/components/FormBuilderSettings/LogicSettings/QuestionRuleInput.tsx

```diff
@@ -80,7 +86,9 @@ export const RuleInput = ({
         logic.rules = [newRule];
       }
       // Update rule variable
-      setValue(name, logic);
+      const required =
+        logic.rules && logic.rules.length > 0 ? true : field.required;
```

Comment:
```suggestion
      const required = logic.rules && logic.rules.length > 0 && field.required;
```

Reply:
I think this will evaluate to the incorrect value in the case where we have a field which we've set to "not required". If we have an non-required field which we then set logic rules on, we want to set required to True. But if the original value of field.required is False, then `logic.rules && logic.rules.length > 0 && field.required` will evaluate to False, when we could like it to evaluate to True :thinking: Or maybe I'm thinking about this wrong?

Reply:
You are right. Forget my comment.